plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/KVMStorageProcessor.java

```diff
@@ -1639,6 +1673,74 @@ public Answer createVolumeFromSnapshot(final CopyCommand cmd) {
         }
     }
 
+    private KVMPhysicalDisk createRBDvolumeFromRBDSnapshot(KVMPhysicalDisk volume, String snapshotName, String name,
+            PhysicalDiskFormat format, long size, KVMStoragePool destPool, int timeout) {
+
+        KVMStoragePool srcPool = volume.getPool();
+        KVMPhysicalDisk disk = null;
+        String newUuid = name;
+        int rbdFeatures = 61;
```

Comment:
Minor: if this is the same as `LibvirtStorageAdaptor.rbdFeatures`, can we expose it and use it instead of a new variable on this method?